src/components/MovieShowPage.js

```diff
@@ -12,58 +12,79 @@ export class MovieShowPage extends Component {
+          const newRatings = { ...user, ratings: ratingData.ratings };
+          updateUser(newRatings);
+        });
       })
-    }).catch(err => console.log('.catch() error on update'))
+      .catch(error => {
+        this.props.handleError(
+          'Problem updating your rating, please try again later.'
+        );
+      });
   };
 
   render() {
     const {
-        id,
-        title,
-        backdrop_path,
-        release_date,
-        overview,
-        average_rating,
-        isLoggedIn,
-        updateUser,
-        user
-      } = this.props;
+      id,
+      title,
+      backdrop_path,
+      release_date,
+      overview,
+      average_rating,
+      isLoggedIn,
+      updateUser,
+      user,
+      errorMessage
+    } = this.props;
 
     return (
       <div className='movie-page'>
         <img className='backdrop' src={backdrop_path} alt={title} />
         <h1 className='movie__title--two'>{title}</h1>
         <p className='in-theaters'>In Theaters: {release_date}</p>
         <p className='overview'>{overview}</p>
-        <p className='average__rating--two'>Average Rating: {Math.round(
-          average_rating * 10 ) / 10}</p>
-        {isLoggedIn &&  (findRating(id, user, 'rating') !== '...') &&
+        <p className='average__rating--two'>
+          Average Rating: {Math.round(average_rating * 10) / 10}
+        </p>
+        {isLoggedIn && findRating(id, user, 'rating') !== '...' && (
           <div className='already-rated'>
-            <p className={findRating(id, user, 'rating') === 10 ? 'top-marks' :
-              'my-rating'}>
+            <p
+              className={
+                findRating(id, user, 'rating') === 10
+                  ? 'top-marks'
+                  : 'my-rating'
+              }>
               My Rating: {findRating(id, user, 'rating')}
             </p>
-            <button className='reset-rating'
-              onClick={() => {deleteRating(findRating(id, user, 'id'), user.id)
-                .then(data => {
-                  fetchRatings(user.id).then(ratingData => {
-                    const updatedRatings = { ...user,
-                      ratings: ratingData.ratings };
-                    updateUser(updatedRatings);
+            <button
+              className='reset-rating'
+              onClick={() => {
+                deleteRating(findRating(id, user, 'id'), user.id)
+                  .then(data => {
+                    fetchRatings(user.id).then(ratingData => {
+                      const updatedRatings = {
+                        ...user,
+                        ratings: ratingData.ratings
+                      };
+                      updateUser(updatedRatings);
+                    });
                   })
-                }).catch(err => console.log('.catch() error on delete'))
-              }}
-            >Reset Rating</button>
+                  .catch(err => console.log('.catch() error on delete'));
```

Comment:
Could you please add the same error handling to line 76's `.catch()` for the deleteRating before we merge this PR?